offline/offline_tools/matlab_hax/sensor_data_plotting/data_src/2014-11-23-02-11-57/sensors.txt

```diff
@@ -1,361 +0,0 @@
-sensors/imu,2014-11-23 02:11:57,15.36,-30.72,249.86,-10.0,13.67,78.17,-123.2,-3.6,-7.16
```

Comment:
I am fine with us getting rid of it completely. I don't really want to support old log file formats